src/components/EventsPage/EventsPage.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { forwardRef } from 'react';
+
+interface EventsPageProps {
+  // You can add any props that the EventsPage component might need in the future
+}
+const EventsPage = forwardRef<HTMLDivElement, EventsPageProps>((props, ref) => {
+  return (
+    <div ref={ref} id="events-section" className=" h-screen">
```

Comment:
@Hariharan482 Id used here is same as id used in the sideNavigation is this expected ?